hw_recogs.ipynb

```diff
@@ -1441,15 +1433,15 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "dsp.settings.configure(lm=lm)"
+    "dspy.settings.configure(lm=lm)"
    ]
   },
   {
    "cell_type": "markdown",
    "id": "840dbd4b-7652-4bb8-bafb-c38bd752c10e",
    "metadata": {},
    "source": [
-    "### Train examples in DSP format"
+    "### Train examples in DSPy format"
```

Comment:
`dsp.Example` below need to be updated to `dspy.Example`

> "This will convert the train set into a list of `dsp.Example` instances to use for demonstrations:"

Reply:
Thanks!